### DISCUSSION TITLE
Can I use delta on `git add . -p`?

### DISCUSSION THREAD

**Discussion by gabrielforster** [dandavison/delta#1861] [Q&A] [answered]
I would like to know if there is any way to change .gitconfig to use delta on `git add . -p`.

Is it possible?

---
Example: I want to use delta here
![image](https://github.com/user-attachments/assets/c86382e2-e29b-43c8-96b7-2d21f53c7dd0)

**Comment by dandavison** [comment, marked as answer]
Hi @gabrielforster, yes, this works if you have the basic config from https://github.com/dandavison/delta?tab=readme-ov-file#get-started.

```gitconfig
[interactive]
    diffFilter = delta --color-only
```

**Reply by gabrielforster** [reply]
Yeah, it worked just fine! Thanks 🚀
